nntrainer/tensor/tensor_pool.cpp

```diff
@@ -259,6 +259,23 @@ TensorPool::requestSpec &TensorPool::getSourceSpec(const std::string &name) {
   return pool.at(parent_spec_idx);
 }
 
+Tensor *TensorPool::placeholder(const std::string &name, const TensorDim &dim) {
+  /// @todo rename requestExternallyAllocateTensor -> placeholder
+  return requestExternallyAllocateTensor(dim, name);
```

Comment:
I recommend you to either rename from the start itself or add in the comments that `placeholder` is an alias for `requestExternallyAllocateTensor`, ensuring that they follow same semantics.

Reply:
This is to keep PR size small as it is already considerably big, this will be done within upcoming PRs.

Reply:
#1697 handles this matter